packages/mask/background/services/helper/r2d2Fetch.ts

```diff
@@ -42,6 +40,14 @@ function proxiedFetch(url: string, info: Request) {
     return originalFetch(req)
 }
 
+async function contentFetch(url: string, config?: RequestInit) {
+    const fetch =
+        process.env.engine === 'firefox' && process.env.manifest === '2' && typeof content === 'object'
+            ? content.fetch
```

Comment:
Content fetch does not exist in the background. You must call it from the content script.